project/src/index.tsx

```diff
@@ -2,12 +2,11 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './components/app/app';
 
-// const CARDS_AMOUNT = 5;
-// const cards = new Array(CARDS_AMOUNT).fill('').map((element, index) => element = index);
+const CARDS_AMOUNT = 5;
+const cards = new Array(CARDS_AMOUNT).fill('').map((element, index) => element = index);
```

Comment:
Можно вернуть сразу индекс)
```suggestion
const cards = new Array(CARDS_AMOUNT).fill('').map((_, index) => index);
```